.github/workflows/bindings-nodejs-publish.yml

```diff
@@ -115,10 +115,28 @@ jobs:
         run: echo "LIBCLANG_PATH=$((gcm clang).source -replace "clang.exe")" >> $env:GITHUB_ENV
         if: matrix.os == 'windows-2019'
 
+      - name: Install gon (macOS)
+        # https://github.com/mitchellh/gon
+        run: brew install mitchellh/gon/gon
+        if: ${{ startsWith(matrix.os, 'macos') }}
+
       - name: Set deployment target (macOS)
         run: echo "MACOSX_DEPLOYMENT_TARGET=10.13" >> $GITHUB_ENV
         if: matrix.os == 'macos-13'
 
+      - name: Add aarch64 target (macOS)
+        if: ${{ startsWith(matrix.os, 'macos') }}
```

Comment:
i suppose first wedo run-if and the other one if-run
kinda inconsistent

Reply:
Aligned it

Reply:
Take the liberty to close this because you have already approved so I do not have to change the date again